kernel/cooking.ml

```diff
@@ -184,6 +184,17 @@ let lift_univs cb subst auctx0 =
     assert (AUContext.is_empty auctx0);
     subst, (Monomorphic_const ctx)
   | Polymorphic_const auctx ->
+    (** Given a named instance [subst := u₀ ... uₙ₋₁] together with an abstract
```

Comment:
What is a named instance?

Reply:
Just that it contains names rather than bound variables.